src/contrib/persistence/Akka.Persistence.Sql.Linq2Db/Streams/ExtSeq.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Threading.Tasks;
+using Akka.Streams.Dsl;
+using LanguageExt;
+
+namespace Akka.Persistence.Sql.Linq2Db.Streams
+{
+    public static class ExtSeq
+    {
+        public static Sink<TIn, Task<Seq<TIn>>> Seq<TIn>() => Sink.FromGraph(new ExtSeqStage<TIn>());
+    }
```

Comment:
At some point benchmarks indicated Language.Ext's `Seq<A>` was far better for this use case than other immutable structures (mostly around concatenating/etc,) so we are using it here in a custom graph/flow.